.github/workflows/ci.yml

```
@@ -6,16 +6,57 @@ name: CI
 on:
   pull_request:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
+  quick_lint:
+    name: Lint (quick)
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v4
+        with: { fetch-depth: 0 } # Required so we can lint commit messages.
```

I would still prefer use just normal YAML instead of embed JSON in yaml
```suggestion
        with:
          fetch-depth: 0 # Required so we can lint commit messages.
```